serverless.yml

```diff
@@ -13,12 +13,31 @@ provider:
       Action:
         - "ses:SendEmail"
       Resource: "*"
+    - Effect: "Allow"
+      Action:
+        - "s3:CreateBucket"
+      Resource:
+        - arn:aws:s3:::${file(env.${opt:stage}.yml):S3_LOGGING_BUCKET}
```

Comment:
I'm wondering if this permission is necessary for the Lambda function. AFAICT, the bucket should get created by CloudFormation during the run of `sls`.

Reply:
Yes, that's true. I've removed this permission.